README.md

```diff
@@ -1,5 +1,11 @@
-# ppa
-A PPA repository for SecureDNA packages
+# PPA
+A PPA repository for SecureDNA packages.  Currently supports [Ubuntu amd64](https://github.com/SecureDNA/ppa).
```

Comment:
It feels a little odd to have have this link back to itself (I figure this URL is the most likely way for somebody to stumble across this readme).

Reply:
Uh, you're right. :)